CppCoreGuidelines.md

```diff
@@ -3193,9 +3193,9 @@ Informal/non-explicit ranges are a source of errors.
 ##### Note
 
 Ranges are extremely common in C++ code. Typically, they are implicit and their correct use is very hard to ensure.
-In particular, given a pair of arguments `(p, n)` designating an array \[`p`:`p + n`),
+In particular, given a pair of arguments `p, n` designating an array `[p, p + n)`,
```

Comment:
i removed the parentheses around the pair in this example to avoid confusion over the fully open interval notation (a, b). ISO 31-11 would also allow the notation `<a, b>` for tuples/pairs, but since not used anywhere else in the document until now (at least I did not see), I thought this would be a better compromise.